### ISSUE TITLE
AttributeError: NoneType object has no attribute Data

### ISSUE THREAD

**Issue by DragoniteSpam** [blender-to-gmstudio/blender-to-smf#53]
When exporting a model with the SMF option, Blender spits out an error.

It's a pretty simple model so I don't know if that means there's some kind of data value I need to assign that isn't there, or if a blender update broke it or something.

![image](https://github.com/user-attachments/assets/7868e524-6c63-4d4f-8104-83e307fa8b13)
[dragon.zip](https://github.com/user-attachments/files/17892132/dragon.zip)

**Comment by bartteunis** [comment]
Looking at that screenshot the error occurs in a file `smf.py` though all of the export code is now in `export_smf.py` (in which that issue is [covered](https://github.com/blender-to-gmstudio/blender-to-smf/blob/e5e6d220207a792e7c92a7568568ab94241cc8eb/export_smf.py#L301)). Could it be that you've copied some older version of the exporter to Blender 4.3?
I had a try at exporting the model you included in Blender 4.3 using the [latest release](https://github.com/blender-to-gmstudio/blender-to-smf/releases/tag/v0.9.3) of the exporter. Which actually still throws an error, though another one: 
```
RuntimeError: Error: Tangent space computation needs a UV Map, "(null)" not found, aborting
```
Once you assign the mesh a (empty) UV map it exports fine however and it can be loaded fine in the SMF Model Tool.
Though the error with the tangent computation and UV map will need to be handled properly in the exporter.

**Comment by bartteunis** [comment]
The issue above is fixed in the [0.9.4 release](https://github.com/blender-to-gmstudio/blender-to-smf/releases/tag/v0.9.4). After installing this new release from Blender 4.3's Add-on Preferences the animation should export fine now and it can be loaded and played in the SMF Editor, at least from my testing here.

**Comment by DragoniteSpam** [comment]
i didn't get a chance to look at this again until just now, but it seems to be working properly, ty!
